lib/sse.js

```diff
@@ -0,0 +1,40 @@
+const { status, header } = require('node-res')
+
+class SSE {
```

Comment:
This is a good shared util which can be refactored into a separate NPM package to be also used by nuxt core (build indicator)

Reply:
Alternative: Nuxt core can require it from `@nuxt/loading-screen/lib/sse.js`

Reply:
Will do this, I think later on we will benefits to have something like `@nuxt/sse`

Reply:
Right now the build indicator is only using the client-side `EventSource`